stacker/actions/build.py

```diff
@@ -0,0 +1,201 @@
+import logging
+
+from . import base
+from .. import exceptions, util
+from ..plan import Plan
+
+logger = logging.getLogger(__name__)
+
+
+class Action(base.BaseAction):
```

Comment:
Do you think it'd be worth naming the Actions a bit more descriptively?  ie: BuildAction, DestroyAction?